release.py

```diff
@@ -79,27 +79,13 @@ def download_artifacts_github_actions(session, token, run_url):
     return paths
 
 
-def build_github_actions_wheels(token, version):
+def fetch_github_actions_wheels(token, version):
     session = requests.Session()
 
-    response = session.post(
-        "https://api.github.com/repos/pyca/cryptography/actions/workflows/"
-        "wheel-builder.yml/dispatches",
-        headers={
-            "Content-Type": "application/json",
-            "Accept": "application/vnd.github.v3+json",
-            "Authorization": "token {}".format(token),
-        },
-        data=json.dumps({"ref": "main", "inputs": {"version": version}}),
-    )
-    response.raise_for_status()
-
-    # Give it a few seconds for the run to kick off.
-    time.sleep(5)
     response = session.get(
         (
             "https://api.github.com/repos/pyca/cryptography/actions/workflows/"
-            "wheel-builder.yml/runs?event=workflow_dispatch"
+            "wheel-builder.yml/runs?event=create"
```

Comment:
I think this is the right event type but https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows and https://docs.github.com/en/rest/reference/actions#workflow-runs are the docs to check.